Quantum-Control-Applications/Superconducting/Single-Flux-Tunable-Transmon/17_cryoscope_1ns.py

```diff
@@ -108,7 +108,7 @@ def baked_waveform(waveform, pulse_duration):
     for i in range(0, pulse_duration + 1):
         with baking(config, padding_method="right") as b:
             if i == 0:  # Otherwise, the baking will be empty and will not be created
-                wf = [0.0] * 16
+                wf = [0.0] * 16  # So one needs to avoid playing the first data point
```

Comment:
Add in the docstring (prerequisite) to set the flux pulse amplitude to give at least 1 oscillation every 5ns